logging/logging.go

```diff
@@ -5,15 +5,24 @@ import (
 	"io"
 	"io/ioutil"
 	"log"
+	"os"
 	"strings"
 
-	"github.com/hashicorp/go-syslog"
+	gsyslog "github.com/hashicorp/go-syslog"
 	"github.com/hashicorp/logutils"
 )
 
 // Levels are the log levels we respond too.
 var Levels = []logutils.LogLevel{"TRACE", "DEBUG", "INFO", "WARN", "ERR"}
 
+// LogLevel is the log level of all logs in this package
+// it is set by the Setup() function and is shared with all other setup variations (i.e. SetupLocal())
+var LogLevel = ""
+
+const (
+	defaultLogLevel = "INFO"
```

Comment:
The default log level in the config package is using `WARN` ([code](https://github.com/hashicorp/consul-terraform-sync/blob/master/config/config.go#L21)). I'm not necessarily advocating that warn is the right level to default to, but wanted to open up the discussion since it is different. What log level would you suggest for consistency?

Reply:
I think WARN is better you're right, we should receive warnings if things aren't directly setup by default.

I've also moved the setting of the default log level globally to where the default logger is created in NewLogFilter for consistency. Doesn't really make